packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -1732,12 +1732,13 @@ class RenderViewport2 extends RenderViewportBase2<SliverPhysicalContainerParentD
         break;
     }
     assert(effectiveExtent != null);
-    offset.applyViewportDimension(effectiveExtent);
-    return offset.applyContentDimensions(
-             // when updating this, also update similar code in performLayout()
-             math.min(0.0, _minScrollExtent + effectiveExtent * anchor),
-             math.max(0.0, _maxScrollExtent - effectiveExtent * (1.0 - anchor)),
-           );
+    final bool didAcceptViewportDimension = offset.applyViewportDimension(effectiveExtent);
+    final bool didAcceptContentDimension = offset.applyContentDimensions(
+       // when updating this, also update similar code in performLayout()
```

Comment:
add a comment in performResize to point to rereredimensions and saying that you're ignoring the return value, etc